src/pages/search/filters/SearchFiltersSortBy.tsx

```diff
@@ -33,16 +32,18 @@ export default function SearchFiltersSortBy() {
 
   return (
     <div className="flex items-center gap-1">
-      <Button
-        variant="light"
-        onClick={() => setIsDesc(!isDesc)}
-        className={isDesc ? '' : 'rotate-180'}
-        isIconOnly
-      >
-        <ArrowDownIcon />
-      </Button>
+      <Tooltip content="Sort order" placement="bottom">
+        <Button
+          variant="light"
+          className={isDesc ? '' : 'rotate-180'}
+          onClick={() => setIsDesc(!isDesc)}
+          isIconOnly
+        >
+          <ArrowDownIcon />
+        </Button>
+      </Tooltip>
       <Select
-        className="w-[150px]"
+        className="w-[180px]"
```

Comment:
[SUG] why can't this be w-fit or something more related to the content?

Reply:
I tried that at first but after some attempts getting this:
<img width="251" alt="Captura de pantalla 2024-07-07 a la(s) 17 47 30" src="https://github.com/Snuffleevees-Paltanchez/snuffleevees-paltanchez.github.io/assets/62082450/2b71fdc4-ec78-4a9b-8d7d-3860a124e696">
I just gave up XD. Maybe there is some tricks with css (or changing parent component styles) that will achieve solving it in a responsive way but I don't it is worth the time

Reply:
jaja I see, I thought it was easier, but I agree with you